tests/helpers/run_sh_command.py

```diff
@@ -12,7 +14,12 @@ def run_sh_command(command: List[str]):
     """Default method for executing shell commands with pytest and sh package."""
     msg = None
     try:
-        sh.python(command, _out=lambda log: print(log.strip()))
+        with StringIO() as buf:
+            sh.python(command, _out=buf)
+            output = buf.getvalue()
+            # Remove color.
+            output = strip_ansi(output)
+        return output
```

Comment:
I think it's a bad idea to parse the output of the command. Checking exit codes is fine, but parsing output is bad. (Linux programs, in theory, shouldn't output anything with successful execution.)

Reply:
In any case, the better way to accomplish this is to tell the program to not produce colors. Does https://amoffat.github.io/sh/sections/faq.html#why-does-my-command-s-output-have-color fix this instead?

Reply:
It looks like the colored module we have does not honor `_tty_out=False`.

Reply:
Another option is not to use `hydra_colorlog`: `hydra/job_logging=stdout hydra/hydra_logging=hydra_debug`

Reply:
What if you set `NO_COLOR=1`? See: https://github.com/borntyping/python-colorlog/blob/8390e7504612a5e44d9229b5d29dd2ca0c0f6f99/colorlog/formatter.py#LL156C30-L156C38

Reply:
Magic!